script/CommonDeployer.s.sol

```diff
@@ -60,9 +61,9 @@ abstract contract CommonDeployer is Script, JsonRegistry {
         messageProcessor = new MessageProcessor(root, tokenRecoverer, deployer);
 
         gasService = new GasService(maxBatchSize, messageGasLimit);
-        gateway = new Gateway(centrifugeId, root, gasService, deployer);
+        gateway = new Gateway(localCentrifugeId, root, gasService, deployer);
```

Comment:
We're assuming here that another previous code will initialize `localCentrifugeId` before calling `deployCommon()`, creating an "invisible" side effect. Why not pass the `localCentrifugeId` as a parameter to express that relation?

Reply:
Not for external values that come from env values.